templates/flavors/default/kustomization.yaml

```diff
@@ -29,6 +30,7 @@ patches:
         labels:
             cni: cilium
             ccm: linode
+            csi: linode
```

Comment:
If we want to have the linode csi driver installed by default, can we add a note to the docs that this is the case?
```
~~~admonish success title=""
Installed by default
~~~
```

Reply:
Sure